src/operator/grid_generator-inl.h

```diff
@@ -36,12 +36,12 @@ struct GridGeneratorParam : public dmlc::Parameter<GridGeneratorParam> {
     DMLC_DECLARE_FIELD(transform_type)
     .add_enum("affine", grid::kAffine)
     .add_enum("warp", grid::kWarp)
-    .describe("transformation type\n    "
-              "if transformation type is affine, data is affine matrix : (batch, 6)\n    "
-              "if transformation type is warp, data is optical flow : (batch, 2, h, w)");
+    .describe("The type of transformation. For `affine`, input data should be an affine matrix "

```

Comment:
I here assume reader has backgrounds about common matrices .
I will attach more description for affine transform , but as for optical flow, detail explanation can take pages, personally prefer a link to wiki.